src/ResourceManager/Compute/Commands.Compute/Models/AzureDiskEncryptionStatusContext.cs

```diff
@@ -3,16 +3,17 @@
 
```

Comment:
@hyonholee, This file does not contain license header. Please add in the nearest PR.